RELEASE.md

```diff
@@ -1,6 +1,15 @@
+# Release 0.3.1
+- Fix compatibility with Julia 0.6 [#341, #330, #293]
+- Support of Stan interface [#343, #326]
+- Fix Binomial distribution for gradients. [#311]
+- Stochastic gradient Hamiltonian Monte Carlo [#201]
+- Disable adaptive resampling for CSMC [#357]
+- Fix resampler for SMC [#338]
+- Initial implementation of Interactive PMCMC [#334]
```

Comment:
why initial ?

Reply:
perhaps I can drop this word -  since it was not included for other new samplers anyway.

Reply:
👍